shared/common-adapters/usernames.js

```diff
@@ -72,6 +72,23 @@ const nonInlineStyle = {
 }
 const inlineProps = isMobile ? {lineClamp: 1} : {}
 
+// TODO: team and channels
+function matchesFilter(name: string, filter: string): boolean {
+  if (!filter) {
+    return true
+  }
+
+  // No need to worry about Unicode issues with toLowerCase(), since
+  // names can only be ASCII.
+  return name.toLowerCase().indexOf(filter.toLowerCase()) >= 0
+}
+
+const sortNamesWithFilter = (names: Array<string>, filter: ?string): Array<string> => {
```

Comment:
i don't think we change the sorting at all, see the design: https://zpl.io/aRMyk0V
search is 'malg' we don't change the tlf order though